rq_dashboard/cli.py

```diff
@@ -138,10 +138,6 @@ def make_flask_app(config, username, password, url_prefix, compatibility_mode=Tr
     multiple=True,
     help="Append specified directories to sys.path",
 )
-@click.option(
-    "--web-background", default="black", help="Background of the web interface"
-)
-@click.option("--delete-jobs", default=False, help="Delete jobs instead of cancel")
```

Comment:
Please keep this configuration options. Just add a warning If this option is specified, we will remove It in next versions.

Reply:
Now it should be ok